test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs

```diff
@@ -57,14 +58,42 @@ public class VersionUtilityFacts
         [InlineData("dnxcore50", "portable-net45+win8", true)]
         [InlineData("dnxcore50", "portable-net451+win81", true)]
         [InlineData("dnxcore50", "portable-net40+sl5+win8", false)]
-        public void FrameworksAreCompatible(string projectTargetFramework, string packageTargetFramework, bool compatible)
+
+        // Core50
+        [InlineData("core50", "core50", true)]
```

Comment:
I think we need to test the inverse as well. Where the project is portable and the package is core50.